payments/payments.go

```diff
@@ -280,14 +280,17 @@ type (
 	}
 
 	RiskRequest struct {
-		Enabled bool `json:"enabled"`
+		Enabled         bool   `json:"enabled"`
+		DeviceSessionId string `json:"device_session_id,omitempty"`
 	}
 
 	PaymentRecipient struct {
-		DateOfBirth   string `json:"dob,omitempty"`
-		AccountNumber string `json:"account_number,omitempty"`
-		Zip           string `json:"zip,omitempty"`
-		LastName      string `json:"last_name,omitempty"`
+		DateOfBirth   string          `json:"dob,omitempty"`
+		AccountNumber string          `json:"account_number,omitempty"`
+		CountryCode   *common.Country `json:"country_code,omitempty"`
```

Comment:
I don't think this should not be a pointer

Reply:
👍